### PR TITLE
Final mobile version,

### PR DESCRIPTION
**Pull request by sergiogval**
Hi, CR! I'd like to stress that I worked on solo all the week, my partner had a personal leave and they did not mached me with anyone else. So, you will be se commitments done boy myself and one collegue. On this realise I implemented the full form functionality, as well as some hover effects at button & text inputs. You will notice that there is a CSS file that is no still on use. I'm trying to diferent files and call the according one with a meta tag on HTML. LLet me know if you have anty questions. I'm already have finiched the web version. I'll request a pull before the day ends.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,26 +14,26 @@
 
   <body>
     <!--HEADER & NAV-->
-    <header class="container">
-      <div class="flex-container3">
+    <header>
```
Comment (barackm) [inline index.html:17]: - [ ] You have done a great job adding validation to your inputs, however, for now your contact form doesn't seem to look like the one in the Figma design, see the difference below 👇 
- Your design
image

- Figma design
image

As you can see:
- [ ] Your inputs are not taking the appropriate width
- [ ] The get in touch button is not the same as the one in Figma
- [ ] The font-size and font-family are not the same as in the Figma design
- [ ] There is an extra empty space at the right of your website
- [ ] It seems that you have forgotten to validate the `<textarea>`, people can submit the form without the message.
- [ ] Please let's add the button interactions such as hover, active (when it's pressed), disabled and enable (default state) are styled as shown in the Figma template 👇 
image


Can you please fix these issues 👍

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @sergiogval, 👋 

You have done an amazing job so far! 👏👏👏
However, there are some issues that you still need to work on to go to the next project. But remember you are almost there 😉  😉 !

# STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: stylem.css [hunk 2]
```diff
@@ -501,8 +466,6 @@ fieldset {
   justify-self: center;
   align-content: center;
   order: 1;
-  width: 327px;
-  height: 48px;
   outline: none;
```
Comment (sergiogval) [inline stylem.css:469]: Change size of email input component

File: stylem.css [hunk 3]
```diff
@@ -514,8 +477,6 @@ fieldset {
   border-radius: 8px;
   justify-self: center;
   order: 1;
```
Comment (sergiogval) [inline stylem.css:479]: Change the size of email input component.

File: stylem.css [hunk 4]
```diff
@@ -527,8 +488,6 @@ fieldset {
   box-sizing: border-box;
   justify-self: center;
   order: 1;
-  width: 327px;
-  height: 114px;
   outline: none;
```
Comment (sergiogval) [inline stylem.css:491]: Change the size of comments component.

File: stylem.css [hunk 5]
```diff
@@ -538,8 +497,6 @@ fieldset {
   position: static;
   display: flex;
   color: white;
```
Comment (sergiogval) [inline stylem.css:499]: Change the form of submit button component.

File: stylem.css [hunk 6]
```diff
@@ -253,7 +240,6 @@ p.text3 {
 }
 
 img.imgabout {
-  width: 80px;
   justify-self: center;
   margin-top: 20px;
   box-shadow: 5px 5px rgba(20, 25, 77, 0.08);
```
Comment (sergiogval) [inline stylem.css:245]: Change for a circle figure of the image, before it was a square.

File: stylem.css [hunk 7]
```diff
@@ -42,6 +42,14 @@ body {
...
+.item {
+  background-image: url("./images/Untitled.png");
+  background-size: auto;
+  border-radius: 8px;
+  box-sizing: content-box;
+  width: auto;
+}
+
 button:hover,
 input.submit:hover {
```
Comment (sergiogval) [inline stylem.css:54]: Add a new class of text for the header of of cards on the portfolio section.

File: index.html [hunk 8]
```diff
@@ -95,7 +95,7 @@ <h2 class="text3">My Recent Works</h2>
         <hr class="separador" />
       </div>
       <div class="item5">
-        <img src="#" alt="IoT" class="item" />
+        <img src="./images/Untitled.png" alt="IoT" class="item" />
         <h2 class="text1">Multi-post Stories Gain+Glory</h2>
```
Comment (sergiogval) [inline index.html:99]: Changed element and class to match the template.

File: stylem.css [hunk 9]
```diff
@@ -344,8 +336,6 @@ li.tab5 {
   display: grid;
   grid-column: 1;
   grid-row: 2;
```
Comment (sergiogval) [inline stylem.css:338]: Recuerda cambiar esto en el archivo para web.

File: stylem.css [hunk 10]
```diff
@@ -298,18 +302,12 @@ li.tab5 {
...
   top: 0;
   bottom: 0;
   list-style: none;
-
-  /* Theme 2 / Neutral 10 */
   background: #ebf0ee;
-
-  /* Neutrals/N50 */
   border: solid 1px #c1c7d0;
   box-sizing: border-box;
```
Comment (sergiogval) [inline stylem.css:312]: Change on item1,item2 & item3 to match the template.

**Review by barackm — CHANGES_REQUESTED** [review]
Hi @sergiogval, 👋 

Great work on making the changes requested by a previous reviewer 👏 👏👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. 😉  😉 

 # STATUS: Required Changes ♻️
### _Check the comments under the review._ 👇 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 11]
```diff
@@ -251,7 +249,7 @@ <h2 class="skills">Skills</h2>
               name="email"
               id="email"
               class="email"
-              placeholder="Email Address"
+              placeholder="Email address"
               required=""
             />
             <label for="comments"></label>
```
[the remark this answers is not in the document]
Comment (sergiogval) [reply index.html:255]: image

[the remark this answers is not in the document]
Comment (sergiogval) [reply index.html:255]: ok I've just ran a "nxp hint ." command locally and it gave me no errors. So I'll a commit and will require the review again. The optional comments, will be consider for the desktop version commit.

**Review by safafa — APPROVED** [review]
## STATUS: APPROVED 🟢 👏
Hello, @sergiogval  great job working on the requested changes 👍 💯

Now you can merge this branch 💪 😄
HAPPY CODING!! AND GOOD LUCK WITH THE NEXT ONES 🤞 🍀

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**